game-app/game-core/src/main/java/tools/image/DecorationPlacer.java

```diff
@@ -679,7 +680,7 @@ private void loadImageFolder() {
           || !imageFolder.getFile().exists()) {
         currentImageFolderLocation = null;
       } else {
-        currentImageFolderLocation = imageFolder.getFile();
+        currentImageFolderLocation = imageFolder.getFile().toPath();
```

Comment:
Any reason to not change the 'getFile' method on FileSave to be 'getPath'?

Reply:
Will do in a future PR, didn't want to make the diff any larger than it already is